docker-compose.yaml

```
@@ -0,0 +1,18 @@
+version: '2.4'
+
+services:
+
+  docker1.webhook.healthchain.cr:
+    image: soinsalud/dockerhub-slack-webhook:latest
+    container_name: docker1.webhook.healthchain.cr
+    hostname: docker1.webhook.healthchain.cr
+    working_dir: "/app"
+    environment:
+      - SLACK_WEBHOOK_URL=${SLACK_WEBHOOK_URL}
```

```suggestion
      - SLACK_WEBHOOK_URL
```

`=${SLACK_WEBHOOK_URL}` is needless

c.f. https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers